sql/moz-fx-data-shared-prod/telemetry/event_types_update_v1/metadata.yaml

```diff
@@ -0,0 +1,17 @@
+---
+friendly_name: telemetry Event Types
+description: Recreate a view on the most recent data from event_types_v1
+owners:
+  - frank@mozilla.com
+labels:
+  application: telemetry
+  incremental: false
+  schedule: daily
+scheduling:
+  dag_name: bqetl_event_rollup
+  destination_table: null
+  query_file_path: 'sql/moz-fx-data-shared-prod/telemetry
+  /event_types/query.sql'
```

Comment:
This is probably incorrect with the linebreak.

Reply:
It is odd, but is still parseable yaml. I think the yamllinter recommended it. I would prefer it on a newline with `>`, let me see if it complains with that approach.

Reply:
Based on the generated output, it looks like it collapses to a single space, which leads to an invalid path.